projects/aws/eks-anywhere/docker/linux/Dockerfile

```
@@ -5,6 +5,8 @@ ARG TARGETARCH
 ARG TARGETOS
 
 RUN yum install tar -y && \
+    yum install nmap -y && \
```

You can input multiple package names to the same install command, as follows:
```suggestion
    yum install -y iputils nmap tar && \
```